components/brave_wallet/browser/internal/hd_key_unittest.cc

```diff
@@ -33,10 +36,18 @@ std::string GetHexAddr(const HDKey* key) {
   return addr.ToHex();
 }
 
+std::string GetWifPrivateKey(std::vector<uint8_t> private_key) {
```

Comment:
`GetWifPrivateKeyForMainnet` since we hardcoded 0x80 here

Reply:
Added testnet flag
https://github.com/brave/brave-core/pull/17978/commits/b6128cbaba4bf9dab4160c02090e0fe7fca1cf2e